src/Sulu/Bundle/SnippetBundle/Resources/translations/admin.de.json

```
@@ -8,5 +8,14 @@
     "sulu_snippet.default_snippets": "Standard Schnipsel",
     "sulu_snippet.snippet_area": "Typ oder Bereich",
     "sulu_snippet.taxonomies": "Taxonomien",
-    "sulu_snippet.webspace_default_snippet": "%webspaceKey% Standard Schnipsel"
+    "sulu_snippet.webspace_default_snippet": "%webspaceKey% Standard Schnipsel",
+    "sulu_activity.text.snippets.created": "{userFullName} hat das Schnipsel \"{resourceTitle}\" erstellt",
+    "sulu_activity.text.snippets.modified": "{userFullName} hat das Schnipsel \"{resourceTitle}\" geändert",
+    "sulu_activity.text.snippets.copied": "{userFullName} hat das Schnipsel \"{resourceTitle}\" kopiert",
+    "sulu_activity.text.snippets.removed": "{userFullName} hat das Schnipsel \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.snippets.translation_added": "{userFullName} hat eine neue Sprachvariante für \"{resourceLocale}\" zum Schnipsel \"{resourceTitle}\" hinzugefügt",
+    "sulu_activity.text.snippets.translation_copied": "{userFullName} hat die Sprachvariante für \"{context_sourceLocale}\" des Schnipsels \"{resourceTitle}\" nach \"{resourceLocale}\" kopiert",
+    "sulu_activity.text.snippets.translation_removed": "{userFullName} hat die Sprachvariante für \"{resourceLocale}\" des Schnipsels \"{resourceTitle}\" gelöscht",
+    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspaces \"{resourceTitle}\" geändert",
+    "sulu_activity.text.webspaces.default_snippet_removed": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspaces \"{resourceTitle}\" gelöscht"
```

```suggestion
    "sulu_activity.text.webspaces.default_snippet_modified": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspace \"{resourceTitle}\" geändert",
    "sulu_activity.text.webspaces.default_snippet_removed": "{userFullName} has das Standardschnipsel für den Bereich \"{context_snippetAreaKey}\" des Webspace \"{resourceTitle}\" gelöscht"
```

I'm not sure about this change

---

would prefer https://github.com/sulu/sulu/pull/6015#discussion_r629892709